docs/sql/statements/create-repository.rst

```diff
@@ -678,6 +678,15 @@ Parameters
  Endpoint root url, only used for testing to connect to an alternative storage
  provider.
 
+.. _sql-create-repo-gcs-token_uri:
+
+**token_uri**
+ | *Type:* ``text``
+ | *Required:* ``false``
+
+ Endpoint oauth token URI, only used for testing to connect to an alternative
+ oauth provider.
```

Comment:
Should this be "Only use this for testing" ?  For public documentation it imho doesn't make sense to tell our users how we use this setting.